CalibTracker/SiStripChannelGain/test/PCL/step3_ALCA.py

```diff
@@ -20,25 +20,39 @@
 #process.load('Configuration.StandardSequences.Reconstruction_cff')
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
```

Comment:
@mmusich

Done!

> On 07 Mar 2016, at 09:45, Marco Musich <notifications@github.com> wrote:
> 
> In CalibTracker/SiStripChannelGain/test/PCL/step3_ALCA.py <https://github.com/cms-sw/cmssw/pull/13609#discussion_r55174775>:
> 
> > @@ -20,25 +20,39 @@
> >  #process.load('Configuration.StandardSequences.Reconstruction_cff')
> >  process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
> This is not necessary anymore since 75X, can you please switch to Configuration.StandardSequences.FrontierConditions_GlobalTag_cff?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/cms-sw/cmssw/pull/13609/files#r55174775>.
>